odoo/tools/convert.py

```diff
@@ -588,6 +588,10 @@ def _tag_record(self, rec, extra_vals=None):
             res[f_name] = f_val
         if extra_vals:
             res.update(extra_vals)
+        if 'sequence' not in res and 'sequence' in model._fields:
+            sequence = self.next_sequence()
+            if sequence:
+                res['sequence'] = sequence
```

Comment:
If you don't set `auto_sequence` in the XML file, this code screws up the default value of `sequence`.
Please don't do that, as it screws up business code that might assign a good default value.

Reply:
No, because `next_sequence` will return `None` when `auto_sequence` is not set